temporal-testing/src/main/java/io/temporal/internal/testservice/SelfAdvancingTimerImpl.java

```diff
@@ -23,10 +23,7 @@
 import io.temporal.workflow.Functions;
 import java.sql.Timestamp;
 import java.time.Duration;
-import java.util.Comparator;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.PriorityQueue;
+import java.util.*;
```

Comment:
Nit: Please use single class imports.

Reply:
We dropped this rule, let's go with Idea defaults about imports.